modules/insights-operator-configuring.adoc

```diff
@@ -20,6 +20,7 @@ The table below describes the available configuration attributes:
 |`scaPullDisabled`|Disables the simple content access entitlements download|Boolean|`false`
 |`clusterTransferInterval`|Specifies how often Insights Operator checks OpenShift Cluster Manager for available cluster transfers|Time interval|`24h`
 |`disableInsightsAlerts`|Disables Insights Operator alerts to the cluster Prometheus instance|Boolean|`False`
+|`httpProxy, httpsProxy, noProxy`|Set custom proxy for Insights Operator to override cluster-wide proxy|URL|`no default`
```

Comment:
I would omit the `to override cluster-wide proxy` (as discussed in Slack)

Reply:
I've made Tomas's change locally.

/label peer-review-needed